src/Directory.Build.targets

```diff
@@ -111,6 +111,18 @@
 
   <Import Project="$([System.IO.Path]::Combine('$(RepoRoot)', 'build', 'GetBuildVersionStub.proj'))" Condition="'$(DisableNerdbankVersioning)' == 'true'" />
 
+  <!-- In circumstances where we aren't using GitVersioning, set
+       versions manually to something vaguely correct. -->
+  <Target Name="OverrideRepoToolsetVersions"
+          AfterTargets="_InitializeAssemblyVersion"
+          Condition="'$(DisableNerdbankVersioning)' == 'true'">
+    <PropertyGroup>
+      <AssemblyVersion>15.1.0.0</AssemblyVersion>
+      <FileVersion>15.8.0.0</FileVersion>
```

Comment:
Should these be populated from https://github.com/Microsoft/msbuild/blob/master/version.json